migrations/2019_11_19_000000_update_social_provider_users_table.php

```diff
@@ -13,18 +13,32 @@ class UpdateSocialProviderUsersTable extends Migration
      */
     public function up()
     {
-        Schema::table('users', function (Blueprint $table) {
-            if (! Schema::hasColumn('users', 'avatar')) {
+        if (! Schema::hasTable('users')) {
+            Schema::create('users', function (Blueprint $table) {
```

Comment:
We should not create the users table as that is not something this package's scope. If your migration has a post date you can just fix the timestamp in the users migration. Usually that is the first migration that should run, that is why this package asumes that the table already exists.